demo/DemoArticles/ArticleRef.js

```diff
@@ -0,0 +1,25 @@
+/*
+#!/usr/bin/env node
+*/
+/* eslint-disable require-jsdoc */
+
+class ArticleRef {
+  constructor(lang, title, revision) {
+    this.lang = lang
```

Comment:
I like the way you've restructured this as a type. Consider indicating that these variables are private by using a leading underscore.